### PR TITLE
feat: add event analytics [FTL-8080]

### PR DESCRIPTION
**Pull request by YoussefAWasfy**
Closes [FTL-8080]

[FTL-8080]: https://lemmatree.atlassian.net/browse/FTL-8080?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: scripts/generate-api.cjs [hunk 1]
```diff
@@ -6,6 +6,14 @@ const SCHEMA_MANAGER_URL = 'https://affinidi-schema-manager.prod.affinity-projec
 const IAM_URL = 'https://affinidi-iam.prod.affinity-project.org'
 const ISSUANCE_URL = 'https://console-vc-issuance.prod.affinity-project.org'
 const VERIFY_URL = 'https://affinity-verifier.prod.affinity-project.org'
+const ANALYTICS_URL = 'https://analytics-stream.staging.affinity-project.org'
```
Comment (JonasEtzold) [inline scripts/generate-api.cjs:9]: Shouldn't we use the prod URL here as well?

Comment (YoussefAWasfy) [reply scripts/generate-api.cjs:9]: the prod api is hidden, this is just to generate http client but the events are sent to prod

File: src/commands/verify-vc.ts [hunk 2]
```diff
@@ -26,12 +29,23 @@ export default class VerifyVc extends Command {
...
     const { flags } = await this.parse(VerifyVc)
-
+    const session = getSession()
     const apiKey = vaultService.get(VAULT_KEYS.projectAPIKey)
 
     const credentialData = await fs.readFile(flags.data, 'utf-8')
     const verifyCredentialInput: VerifyCredentialInput = JSON.parse(credentialData)
     const verification = await verfierService.verifyVC(apiKey, verifyCredentialInput)
+    const analyticsData: EventDTO = {
+      name: 'VC Verified',
```
Comment (JonasEtzold) [inline src/commands/verify-vc.ts:39]: isn't this missing a `_` ?

Comment (JonasEtzold) [reply src/commands/verify-vc.ts:39]: Ah forget it...saw the definition now.

**Review by JonasEtzold — APPROVED** [review]
Good work and good start @YoussefAWasfy. I only wonder, if the analytics call could be put into a separate function which is than only called with the dynamic part like the event name or meta data. This would reduce some of the code repetition in all of the command files. But this can be also considered at a later code iteration.

**Pull request merged**